ui/server/src/main/resources/defaultUiConfig.conf

```diff
@@ -86,4 +86,86 @@ authentication: {
     Accept: ${?AUTHENTICATION_HEADERS_ACCEPT}
   }
   usersFile: ${?AUTHENTICATION_USERS_FILE}
-}
\ No newline at end of file
+}
+
+processToolbarConfig {
+  defaultConfig {
+    "top-left": [
+      { type: "tips-panel" }
+      { type: "creator-panel" }
+      { type: "versions-panel" }
+      { type: "comments-panel" }
+      { type: "attachments-panel" }
+    ]
+    "top-right": [
+      {
+        type: "process-info-panel"
+        buttons: [
+          { type: "process-save", disabled: { archived: true } }
+          { type: "process-deploy", disabled: { subprocess: true, archived: true, type: "oneof" } }
+          { type: "process-cancel", disabled: { subprocess: true, archived: true, type: "oneof" } }
+          { type: "custom-link", title: "metrics", icon: "/assets/buttons/metrics.svg", templateHref: "/metrics/{{processName}}", disabled: { subprocess: true } }
+        ]
+      }
+      {
+        id: "view-panel"
+        type: "buttons-panel"
+        buttons: [
+          { type: "view-business-view" }
+          { type: "view-zoom-in" }
+          { type: "view-zoom-out" }
+          { type: "view-reset" }
+        ]
+      }
+      {
+        id: "edit-panel"
+        type: "buttons-panel"
+        hide: { archived: true }
+        buttonsVariant: "small"
+        buttons: [
+          { type: "edit-undo" }
+          { type: "edit-redo" }
+          { type: "edit-copy" }
+          { type: "edit-paste" }
+          { type: "edit-delete" }
+          { type: "edit-layout" }
+        ]
+      }
+      {
+        id: "process-panel"
+        type: "buttons-panel"
+        buttons: [
+          { type: "process-properties", hide: { subprocess: true } }
+          { type: "process-compare" }
+          { type: "process-migrate", hide: { archived: true }, disabled: { subprocess: true } }
```

Comment:
hide and disabled seem a bit inconsistent? Hidden/disabled or hide/disable?

Reply:
why migrate is disabled for subprocess?

Reply:
Changed